examples/aws-testing/cluster.lokocfg

```diff
@@ -33,13 +33,41 @@ cluster "aws" {
   //os_channel       = "stable"
   //os_version       = "current"
 
+  conntrack_max_per_core = 0
```

Comment:
What's the reasoning for making the example more complicated? That we might use it later in tests?

Reply:
ah, it perhaps shouldn't be here, good catch.

Reply:
:facepalm: this is of course mistakenly commited when I was testing based on example configs. It shouldn't be committed.